platforms/core-runtime/internal-instrumentation-processor/build.gradle.kts

```diff
@@ -42,6 +42,7 @@ dependencies {
     implementation(libs.jacksonDatabind)
 
     implementation(project(":base-services"))
+    implementation(project(":logging"))
```

Comment:
❓ Why is this necessary? Is this because of the deprecation logger?

Reply:
Removed, we reference `DeprecationLogger`, but we actually don't need to reference the type, we can just use a string.